src/UserAdmin/src/Smart.FA.Catalog.Web/Identity/UserIdentity.cs

```diff
@@ -19,7 +20,10 @@ public class UserIdentity : IUserIdentity
     public Trainer CurrentTrainer => Identity?.Trainer!;
 
     /// <inheritdoc />
-    public bool IsSuperUser => _httpContextAccessor.HttpContext!.User.IsInRole("SuperUser");
+    public bool IsSuperUser => _httpContextAccessor.HttpContext!.User.IsInRole(Roles.SuperUser);
+
+    /// <inheritdoc />
+    public bool IsSocialMember => _httpContextAccessor.HttpContext!.User.IsInRole(Roles.SocialMember);
```

Comment:
Do we need the word `Social`?
What purpose does it serve?
```suggestion
    public bool IsMember => _httpContextAccessor.HttpContext!.User.IsInRole(Roles.Member);
```

Reply:
Yes it is to distinguish social members from permanent member so the "social" keyword is key to discrimate those two types of users.

Reply:
@maxime-poulain @LievenPraet I have temporarily (before our meeting with Roger) changed that to "Shareholder".